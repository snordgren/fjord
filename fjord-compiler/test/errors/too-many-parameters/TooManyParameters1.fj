module test.errors.TooManyParameters1

myFun : Int -> Int
myFun x y = 
  x
