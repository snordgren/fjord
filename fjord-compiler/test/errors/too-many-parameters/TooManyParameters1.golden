errors/too-many-parameters/TooManyParameters1.fj:4:9:
  |
4 | myFun x y = 
  |         ^^^^^
too many parameters, expected 1
